db/osqlblockproc.c

```diff
@@ -1297,7 +1297,8 @@ static int apply_changes(struct ireq *iq, blocksql_tran_t *tran, void *iq_tran,
         if (out_rc)
             break;
     }
-
+#if 0
```

Comment:
I wanted to keep this for reference, since the original author is  not around anymore.  It might come back if we need to have pre-commit callbacks.